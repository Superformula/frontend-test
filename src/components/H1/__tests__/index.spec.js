import React from 'react';
import { mount } from 'enzyme';
import { H1 } from '../';

describe('H1', () => {
  it('should render without failing', () => {
    const wrapper = mount(<H1>Restaurants</H1>);
    expect(wrapper).toMatchSnapshot();
  });
});
