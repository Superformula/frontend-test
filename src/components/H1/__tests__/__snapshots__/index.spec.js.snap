// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should render without failing 1`] = `
.c0 {
  color: #333;
  font-weight: 300;
  font-size: 32px;
  line-height: 40px;
  margin-top: 42px;
  margin-bottom: 9px;
}

@media (min-width:758px) {
  .c0 {
    font-size: 54px;
    line-height: 64px;
    -webkit-letter-spacing: .96px;
    -moz-letter-spacing: .96px;
    -ms-letter-spacing: .96px;
    letter-spacing: .96px;
    margin-top: 36px;
    margin-bottom: 24px;
  }
}

<H1>
  <h1
    className="c0"
  >
    Restaurants
  </h1>
</H1>
`;
